src/message/fetchActions.js

```diff
@@ -43,6 +44,16 @@ const messageFetchStart = (narrow: Narrow, numBefore: number, numAfter: number):
   numAfter,
 });
 
+/* eslint-disable-next-line no-unused-vars */
+const messageFetchError = (args: {| narrow: Narrow, error: Error |}): Action => {
```

Comment:
Is this eslint-disable needed? It seems like not -- perhaps it dates from an earlier draft.

Reply:
Ah—in the commit that adds it, it isn't used, so the ignore is needed there. But then when we start using it, it should disappear; I'll do that.